apps/ccusage/README.md

```
@@ -118,12 +121,53 @@ npx ccusage --compact  # Force compact table mode
+```bash
+# Show prompts per 5-hour block
+npx ccusage blocks --prompts
+
+# Combined with other flags
+npx ccusage blocks --prompts --recent  # Last 3 days
+npx ccusage blocks --prompts --json    # JSON output
+```
+
+### Example Output
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)